yesod-test/Yesod/Test/Internal.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | This module exposes functions that are used internally by yesod-test.
+-- The functions exposed here are _not_ a stable API—they may be changed or removed without any major version bump.
+--
+-- That said, you may find them useful if your application can accept API breakage.
+module Yesod.Test.Internal
+  ( getBodyTextPreview
+  , contentTypeHeaderIsUtf8
+  , assumedUTF8ContentTypes
```

Comment:
I thought it was plausible people would want these for any custom assertion functions they'd write. But, they don't feel like something that should be part of the stable API

Reply:
But, I'm pretty flexible on this and don't have much experience with releasing Internal modules